common-util/formatDate.js

```diff
@@ -0,0 +1,10 @@
+export const formatDate = (date) => {
+  const newDate = new Date(date);
+  const options = {
+    day: '2-digit',
+    month: 'long',
+    year: '2-digit',
+  };
+  const formattedDate = newDate.toLocaleDateString('en-GB', options);
```

Comment:
```suggestion
  const formattedDate = newDate.toLocaleDateString('en-US', options);
```
as we use American english, afaik the order of the day and month is different

Reply:
Mmm I think GB works better as it displays it as 11-Nov-24 instead of Nov-28,-24

Reply:
the replace call below makes this weird

would remove the `-` replacement, and set as en-US, not 100% on product preference, but US > GB in market share